### DISCUSSION TITLE
Display menu on top or bottom depending on the screen offset

### DISCUSSION THREAD

**Discussion by MrPhyaeSoneThwim** [tailwindlabs/headlessui#498] [Help]
Hey can you please help me how to position Menu.Items depending on the screen offset because I can’t position the items on top when the menu is near to the bottom of the browser.

**Comment by RobinMalfait** [comment]
Hey! I would recommend you to use something like Popper (https://popper.js.org). Here is an example: https://github.com/tailwindlabs/headlessui/blob/91007b75efdec6500b1afb27d7946b1488eb80c6/packages/%40headlessui-react/pages/menu/menu-with-popper.tsx

**Reply by MrPhyaeSoneThwim** [reply]
Yeah it works for me. Really appreciate for your great help..
